noir/noir-repo/noir_stdlib/src/grumpkin_scalar_mul.nr

```diff
@@ -1,7 +1,12 @@
 use crate::grumpkin_scalar::GrumpkinScalar;
-use crate::scalar_mul::fixed_base_embedded_curve;
+use crate::scalar_mul::{fixed_base_embedded_curve, variable_base_embedded_curve};
 
 pub fn grumpkin_fixed_base(scalar: GrumpkinScalar) -> [Field; 2] {
     // TODO: this should use both the low and high limbs to do the scalar multiplication
```

Comment:
Both limbs seems to be used (see code [here](https://github.com/AztecProtocol/aztec-packages/blob/98f8a9cd0a035e9bdb1fca6cf5c9c43c40b9a474/noir/noir-repo/acvm-repo/bn254_blackbox_solver/src/fixed_base_scalar_mul.rs#L46)). Can the reviewer sanity check me that it's the case? Will remove the TODOs once confirmed.

Reply:
Decent chance that this TODO is stale from when this wasn't the case.